nvflare/app_common/utils/export_utils.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from nvflare.apis.fl_constant import ConnPropKey, FLMetaKey
+from nvflare.apis.fl_context import FLContext
+from nvflare.fuel.data_event.utils import get_scope_property
+
+
+def update_export_props(props: dict, fl_ctx: FLContext):
+    site_name = fl_ctx.get_identity_name()
+    auth_token = get_scope_property(scope_name=site_name, key=FLMetaKey.AUTH_TOKEN, default="NA")
+    signature = get_scope_property(scope_name=site_name, key=FLMetaKey.AUTH_TOKEN_SIGNATURE, default="NA")
```

Comment:
Why not use None as default?

"if auth_token" returns True with "NA".

Reply:
This is because the auth_token needs to be added to message header and sent out.